### ISSUE TITLE
computer time capsule

### ISSUE THREAD

**Issue by mobeets** [hahayes/hahayes-projects#23]
there's probably way too many platforms/apps/data-types for this, but it would be cool if your computer could compile a list of your most-opened/edited songs/photos/documents/websites each year, serving as a sort of snapshot of what sorts of things you did on your computer at a certain time in your life.

the problem i'm guessing is that people use data in so many different ways. like for music, i listen to music in grooveshark, itunes, and rdio, but i'm assuming there are many more music things other people use, and that's just for music. you'd want this time capsule though to cover as wide a spectrum as possible, _and_ to be automatic (i.e. not requiring the user to make his own time capsule, though that could be cool too. in fact, maybe that's totally acceptable and fun. like the app pops up every week and says "what's your favorite song? drag file here.")

**Comment by mobeets** [comment]
actually, yeah, maybe this doesn't need to be automatic at all. like [lifeslice](http://lifeslice.wanderingstan.com/) (which i've been using for a while now, by the way, and it's completely awesome), only instead of taking a photo it asks you a question.

**Comment by silky** [comment]
yeah; i've seen various things for doing this as long as you're using a certain editor, or something.

there might've been one more recently for browsing as well, but even still you'd need to carefully categorise things.

i think an interactive question approach is probably by far the best. or at least, probably easy to do and customise.

for me personally a terminal app would be cute; something that whenever i open a new terminal once a week it asks me a question. then i could say "later" or answer it, and be done. but i'm not sure many other people would want that?

**Comment by mobeets** [comment]
if you can find a link to the browsing one, i'd be interested to see that.

jess and i were brainstorming for this idea this evening, and we got as far as naming it "these days" since it's not doing anything special, but its main goal is to emphasize that the things you give it are supposed to represent typical things you do _lately_ as opposed to just _now_. also the idea that you should be able to scale the importance of things you tell it.

best thing that would come out of this i think would be automatic best-of lists. i love best-of lists.

**Comment by silky** [comment]
yeah; i agree the "lately" is a bit easier to fill out then "now". otherwise it's just too much information.

if you like best-of lists, check out - https://github.com/jnv/lists

( edit: i can't find the thing i was talking about above. )
